deny.toml

```diff
@@ -58,11 +59,8 @@ multiple-versions = "deny"
 wildcards = "deny"
 
 skip-tree = [
-    { name = "h2", version = "0.3.11" }, 
-    # aws sdk depends on hyper, which pulls in an older version of itoa
-    { name = "hyper", version = "0.14.16" },
```

Comment:
It might be a bit noisy for a commit message, but it's probably a good idea to call it out at least in the PR. In this instance the SDK is no longer pulling those older versions so the exceptions are no longer required.